openshift-deployment.yml

```diff
@@ -52,15 +52,18 @@
         - centos-containers
 
     - name: Create OpenShift routes
-      shell: oc create route edge apidoc --service vmaas-apidoc-{{ app_id }} --port {{ apidoc_container_port }}; oc create route edge reposcan --service vmaas-reposcan-{{ app_id }} --port {{ reposcan_container_port }}; oc create route edge webapp --service vmaas-webapp-{{ app_id }} --port {{ webapp_container_port }}
+      shell: oc create route edge apidoc --service vmaas-apidoc --port {{ apidoc_container_port }}; oc create route edge reposcan --service vmaas-reposcan --port {{ reposcan_container_port }}; oc create route edge webapp --service vmaas-webapp --port {{ webapp_container_port }};
       tags:
         - up
         - routes-up
 
-    - name: Create temporary OpenShift routes
-      shell: oc create route edge apidoc-upgrade --service vmaas-apidoc-{{ app_id }} --port {{ apidoc_container_port }}; oc create route edge reposcan-upgrade --service vmaas-reposcan-{{ app_id }} --port {{ reposcan_container_port }}; oc create route edge webapp-upgrade --service vmaas-webapp-{{ app_id }} --port {{ webapp_container_port }}
+    - name: Create OpenShift upgrade routes
+      shell: oc create route edge apidoc-upgrade --service vmaas-apidoc-upgrade --port {{ apidoc_container_port }}; oc create route edge reposcan-upgrade --service vmaas-reposcan-upgrade --port {{ reposcan_container_port }}; oc create route edge webapp-upgrade --service vmaas-webapp-upgrade --port {{ webapp_container_port }}
       tags:
+        - up
+        - routes-up
```

Comment:
also not when this tag is specified, perhaps routes-up and routes-down tags could be removed completely because they are not much useful